diff --git a/.gitconfig b/.gitconfig
index 93f4669..a05759b 100644
--- a/.gitconfig
+++ b/.gitconfig
@@ -1,6 +1,3 @@
-[user]
-    name = Anthony Clark
-    email = clark.anthony.g@gmail.com
 [color]
     ui = auto
 [color "branch"]
@@ -24,5 +21,3 @@
     up = !git remote update --prune && git merge --ff-only @{u}
     rebaseu = rebase --preserve-merges @{u}
     logu = log -p @{u}..
-[push]
-    default = simple
diff --git a/.vimrc b/.vimrc
index 1c18282..4aa1f17 100644
--- a/.vimrc
+++ b/.vimrc
@@ -139,9 +139,9 @@ endif
 
 " Choose color scheme
 if &term == "linux"
-    colorscheme desert           " VTs
+    colorscheme default          " VTs
 else
-    colorscheme ac               " for other terminals (ie - urxvt)
+    colorscheme default          " for other terminals (ie - urxvt)
 endif
 
 " Map Sync
